packages/gatsby-remark-prismjs/README.md

```diff
@@ -228,6 +228,29 @@ You can also add line highlighting. It adds a span around lines of code with a
 special class `.gatsby-highlight-code-line` that you can target with styles. See
 this README for more info.
 
+To highlight lines, you can use one of the following directives as comments in your
+code:
+
+- `highlight-line` highlights the current line;
+- `highlight-next-line` highlights the next line;
+- `highlight-start` highlights the lines until the matching `hightlight-end`;
+- `highlight-range{1, 4-6}` will highlight the next line, and the fourth, fifth and sixth lines.
```

Comment:
I'm not actually sure this is an improvement! I'd almost rather we cut this out. What do you think?

Reply:
Added it for feature parity with `gatsby-remark-embed-snippet` (see [this bit of docs](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-remark-embed-snippet#yaml-example)), however I reckon it can be quite confusing. Should we remove it from the docs and keep it in the code, so as not to break the embed-snippet plugin?

Reply:
ah! Yeah, I suppose if we're providing the feature we may as well document it!

Thanks for catching that!

Reply:
Adding it back in! Will look for a proper example though, so as to make all directives as clear as possible :)